### PR TITLE
Added CVE-2017-1000215

### PR DESCRIPTION
**Pull request by kurtseifried**

### CODE REVIEW

File: 2017/1000xxx/CVE-2017-1000215.json [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+                        "value": "shell command injection"
+                    }
+                ]
+            }
+        ]
+    },
+    "references": {
+        "reference_data": [
+            {
+                "url": "https://github.com/xrootd/xrootd/blob/v4.6.1/docs/ReleaseNotes.txt"
```
Comment (cve-team) [inline 2017/1000xxx/CVE-2017-1000215.json]: There's nothing in the release notes for 4.6.1 that obviously correlates with shell command injection. Am I not seeing it? Or is there another reference we can use to provide provenance?

**Comment by kurtseifried** [comment]
Email sent to original requestor

**Comment by kurtseifried** [comment]
If the original requestor doesn't reply in say .. 2 weeks I vote we reject this, REJECT:BAD_REF_URL

**Comment by kurtseifried** [comment]
From the original reporter: 

The issue is fixed in https://github.com/xrootd/xrootd/commit/befa2e627a5a33a38c92db3e57c07d8246a24acf.
The issue itself is in https://github.com/xrootd/xrootd/blob/befa2e627a5a33a38c92db3e57c07d8246a24acf/src/XrdSecgsi/XrdSecgsiGMAPFunLDAP.cc#L85, where unescaped user-specified input is used to construct an ldapsearch command passed to popen().

**Comment by kurtseifried** [comment]
do you want me to edit the branch and resubmit?

**Comment by cve-team** [comment]
Thanks, Kurt. Please update the file in this branch and let me know when you're finished.

**Comment by kurtseifried** [comment]
updated

**Pull request merged**